src/inject/GPMInject/interface/toast.js

```diff
@@ -0,0 +1,27 @@
+import { remote } from 'electron';
+
+Emitter.on('pauseAfter:show', () => {
+  const toast = document.createElement('paper-toast');
```

Comment:
Also, general question, is the toast link colored for you? It doesn't seem to be for me, but maybe that's just me...